fracas/rules.cpp

```diff
@@ -234,6 +234,7 @@ static GameDefs g = {
 	0,	// UNDEATH_CONTAGION
 	0,   // REGIONS_ECONOMY
 	0, // ADVANCED_TACTICS
+	0,	// OVERWHELMING
```

Comment:
probably its better to extend comment to explain feature. Otherwise it may be left as //REGIONS_ECONOMY and none actually knows what is it.

Reply:
Description added in main file